utils.js

```diff
@@ -0,0 +1,35 @@
+class RecentlyUsedList {
+
+    constructor() {
+        this.items = [];
+    }
+
+    add(newItem) {
+        if (this.items.indexOf(newItem)) {
+            var position = this.items.indexOf(newItem);
+            var existingItem = this.items[position];
+
+            this.items.splice(position, 1);
+            this.items.splice(0, 0, existingItem);
+        } else {
+            this.items.unshift(newItem);
+        }
+    };
+
+    lenght() {
+        var size = this.items.length;
+        return size;
+    }
+
+    itemat(index) {
```

Comment:
`itemat` isn't a descriptive name. Also, no need a function as this.items[n] is enough.